libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+                                                                        "checkMethod",
+                                                                        "vri",
+                                                                        "identityCheckPolicy",
+                                                                        "published",
+                                                                        "txn",
+                                                                        "24929d38-420c-4ba9-b846-3005ee691e26"),
+                                                                Map.of(
+                                                                        "checkMethod",
+                                                                        "pvr",
+                                                                        "biometricVerificationProcessLevel",
+                                                                        3,
+                                                                        "txn",
+                                                                        "24929d38-420c-4ba9-b846-3005ee691e26")))
+                                                .build()))
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                Instant.ofEpochSecond(1705986521));
+    }
+
+    static String vcF2fBrp() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .name(
+                                List.of(
+                                        Map.of(
+                                                VC_NAME_PARTS,
+                                                List.of(new NameParts("Chris", VC_GIVEN_NAME)))))
+                        .birthDate(List.of(new BirthDate("1984-09-28")))
+                        .passport(null)
+                        .residencePermit(
+                                List.of(
+                                        Map.of(
+                                                "icaoIssuerCode",
+                                                "UTO",
+                                                "documentType",
+                                                "CR",
+                                                "documentNumber",
+                                                "AX66K69P2",
+                                                "expiryDate",
+                                                "2030-07-13")))
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(
+                                List.of(
+                                        TestVc.TestEvidence.builder()
+                                                .txn("some-uuid")
+                                                .type(CRI_STUB_CHECK_EVIDENCE_TYPE)
```

Comment:
I'm not sure what this type should be, but I'm don't think it should be `CriStubCheck`?